src/Tes/Models/TesOutput.cs

```diff
@@ -60,12 +60,6 @@ public TesOutput()
         [DataMember(Name = "type")]
         public TesFileType Type { get; set; }
 
-        /// <summary>
-        /// Gets or Sets Path Prefix
-        /// </summary>
-        [DataMember(Name = "pathPrefix")]
-        public string PathPrefix { get; set; }
```

Comment:
Since this property is in the published TES 1.1 spec, should we instead check that the value (if it exists) is equal to the start of `Path` up to (but no including) the point of the level of the first wildcard and throw a not-supported error if not identical? Otherwise we just ignore it.

Reply:
I understand the point about the spec. However, I see little upside to having the property and not supporting it as defined, albeit its ambiguity. It would not make us 1.1 complaint in any case. I think not having it in the model clearly communicates that we don’t support it, even if it makes our 1.1 support formally incomplete.